packages/ketcher-polymer-editor-react/src/components/shared/ui/Tabs/Tabs.jsx

```diff
@@ -31,40 +30,70 @@ function TabPanel({ children, value, index, ...other }) {
 
 function CustomTabs(props) {
   const [tabIndex, setTabIndex] = useState(0)
-  const { tabs, contentClassName, className } = props
+  const { tabs, contentStyle, tabsStyle } = props
   const tabPanel = tabs[tabIndex]
   const Component = tabPanel?.component
   const componentProps = tabPanel?.props
+  const theme = useTheme()
 
   const handleChange = (event, newTabIndex) => {
     setTabIndex(newTabIndex)
   }
 
+  const styleTabs = {
+    height: 24,
+    minHeight: 24
+  }
+
+  const styleTab = {
+    minHeight: 24,
+    minWidth: 60,
+    height: 1,
+    width: 65,
+    padding: 0,
+    fontSize: 10,
+    cursor: 'pointer',
+    textAlign: 'center',
+    backgroundColor: theme.colors.tab.regular,
+    color: theme.colors.text.black,
+
+    '&:hover': {
+      backgroundColor: theme.colors.tab.hover,
+      color: theme.colors.text.white
+    },
+
+    '&.active': {
+      color: theme.colors.text.white,
+      backgroundColor: theme.colors.tab.active
+    }
+  }
+
+  const styleIndicator = {
+    display: 'none'
+  }
+
   return (
     <>
       <Tabs
         value={tabIndex}
         onChange={handleChange}
-        className={clsx(classes.tabs, className)}
-        sx={{ minHeight: 24 }}
+        sx={{ ...styleTabs, ...tabsStyle }}
```

Comment:
sx is `mui`'s built-in styling prop. I believe we should switch to `css` prop or use `styled` component from emotion.